src/EventLoop.php

```diff
@@ -350,13 +351,31 @@ public static function getDriver(): Driver
     }
 
     /**
-     * Create an object used to suspend and resume execution, either within a fiber or from {main}.
+     * Returns an object used to suspend and resume execution of the current fiber or {main}.
+     *
+     * Calls from the same fiber will return the same suspension object.
+     *
+     * @return Suspension
+     */
+    public static function getSuspension(): Suspension
+    {
+        return self::getDriver()->getSuspension();
+    }
+
+    /**
+     * Returns an object used to suspend and resume execution of the current fiber or {main}.
+     *
+     * Calls from the same fiber will return the same suspension object.
      *
      * @return Suspension
+     *
+     * @deprecated This old name is only kept temporarily to allow smooth transitions from 0.1 to 0.2 and will be
+     *     removed at a later point.
      */
+    #[Deprecated]
```

Comment:
maybe not? 😅 

This doesn't add any benefits over using `@deprecated` tag, PhpStorm can already spot it, and so all other SA tools.

Reply:
Don't really care, I'm fine with either.

Reply:
Let's use `@deprecated` to avoid the dependency then, even if it is just dev.